x-pack/legacy/plugins/siem/server/lib/detection_engine/scripts/signals/root_or_admin_update_1.json

```diff
@@ -12,5 +14,8 @@
   "to": "now-5m",
   "query": "user.name: root",
   "language": "kuery",
-  "references": ["https://update1.example.com", "https://update2.example.com"]
-}
+  "references": [
+    "https://update1.example.com",
+    "https://update2.example.com"
+  ]
+}
```

Comment:
Un-needed change. I would revert this back as prettier would just change it back to the old form and we'll have changes that are just stylist back and forth in the commits would will look weird.